test/integration/features/coolstore.feature

```diff
@@ -0,0 +1,96 @@
+@coolstore
+Feature: Cool Store
+  In order to test Minishift under load
+  I need to setup a test environment
+  Using CentOS and 4G of memory
+  After which I will import image streams
+  Finally I will deploy the Cool Store
+  
+  Scenario: Enabling 'xpaas' add-on
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift addons enable xpaas"
+     Then stdout should contain
+      """
+      Add-on 'xpaas' enabled
+      """
+
+  Scenario: Starting Minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --memory 4096 --iso-url centos" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: User can create new namespace 'coolstore'
+   Given Minishift has state "Running"
+    When executing "oc new-project coolstore" succeeds
+    Then stdout should contain
+     """
+     Now using project "coolstore"
+     """
+  
+  Scenario: User can create a template for coolstore
+   Given Minishift has state "Running"
+    When executing "oc create -f ./templates/coolstore.yaml" succeeds
+    Then stdout should contain
+     """
+     template "coolstore" created
+     """
+
+  Scenario: User can deploy the coolstore template to namespace 'coolstore' 
+   Given Minishift has state "Running"
+    When executing "oc new-app --template=coolstore" succeeds
+    Then stdout should contain
+     """
+     Success
+     """
+
+  Scenario: Successfully deployed services to namespace coolstore
+   Given Minishift has state "Running"
+    When services "web-ui, inventory, catalog, cart, coolstore-gw" rollout successfully
+     And executing "oc status -v" succeeds
+    Then stdout should not contain
+     """
+     has failed
+     """
+
+  Scenario: Should be able to get the pod identifier for the API gateway 
+   Given Minishift has state "Running"
+    When setting scenario variable "coolstore-gw" to the stdout from executing "oc get pods -o name -l application=coolstore-gw" when succeeds
```

Comment:
> Should it not be "setting"? and potentially a "succeeds" in the end?

This is what I had, but this was wrong? Actually I think the succeeds is unnecessary... linguistic it should be 'from' as it indicates the location or state it comes from, while 'of' would indicate the actual value, as it is a fixed given.